### PR TITLE
Feature/vbeck/object string updates

### PR DESCRIPTION
**Pull request by lifewithryan**
# Critical Changes

# Changes
-- updates to object descriptions
# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: src/objects/Address__c.object [hunk 1]
```diff
@@ -514,9 +528,9 @@ MailingPostalCode__c
     </fields>
     <fields>
         <fullName>Seasonal_End_Year__c</fullName>
-        <description>The year when this address no longer replaces the default address.</description>
+        <description>The year that this Address no longer replaces the Default Address.</description>
         <externalId>false</externalId>
-        <inlineHelpText>The year when this address no longer replaces the default address.</inlineHelpText>
+        <inlineHelpText>The year that this Address no longer replaces the Default Dddress.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Address__c.object:533]:
```suggestion
        <inlineHelpText>The year that this Address no longer replaces the Default Address.</inlineHelpText>
```
Looks like a minor typo.

File: src/objects/Attendance_Event__c.object [hunk 2]
```diff
@@ -101,9 +101,9 @@
     <fields>
         <fullName>Contact__c</fullName>
         <deleteConstraint>SetNull</deleteConstraint>
-        <description>The student who is either tardy or absent.</description>
+        <description>The student who's tardy or absent.</description>
```
Comment (MatthewBlanski) [inline src/objects/Attendance_Event__c.object:104]:
```suggestion
        <description>The student who&apos;s tardy or absent.</description>
```
My understanding was that we are using the ampersand notation for these.

File: src/objects/Attendance_Event__c.object [hunk 3]
```diff
@@ -101,9 +101,9 @@
...
         <externalId>false</externalId>
-        <inlineHelpText>The student who is either tardy or absent.</inlineHelpText>
+        <inlineHelpText>The student who's tardy or absent.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Attendance_Event__c.object:106]:
```suggestion
        <inlineHelpText>The student who&apos;s tardy or absent.</inlineHelpText>
```
Here as well

File: src/objects/Course_Enrollment__c.object [hunk 4]
```diff
@@ -83,7 +83,7 @@
     <fields>
         <fullName>Affiliation__c</fullName>
         <deleteConstraint>SetNull</deleteConstraint>
-        <description>The Academic Program associated with this Course Connection. Represents a Contact-Account Affiliation.</description>
+        <description>If this Contact is affiliated with an Academic Program through an existing Program Enrollment record, and that Program Enrollment is related to this Course Connection, this Affiliation field is auto-populated with the Program Enrollment's Affiliation. If both conditions aren't true, then this Affiliation field is left blank.</description>
```
Comment (MatthewBlanski) [inline src/objects/Course_Enrollment__c.object:86]:
```suggestion
        <description>If this Contact is affiliated with an Academic Program through an existing Program Enrollment record, and that Program Enrollment is related to this Course Connection, this Affiliation field is auto-populated with the Program Enrollment's Affiliation. If both conditions aren&apos;t true, then this Affiliation field is left blank.</description>
```
Again with the apostrophe here.

File: src/objects/Course_Enrollment__c.object [hunk 5]
```diff
@@ -158,19 +164,19 @@
...
+        <inlineHelpText>For faculty members, indicates they are the primary faculty (for example, lead instructor) for the Course Offering.</inlineHelpText>
         <label>Primary</label>
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
     <fields>
         <fullName>Program_Enrollment__c</fullName>
         <deleteConstraint>SetNull</deleteConstraint>
-        <description>The Program Enrollment record number for the Program the student is enrolled in. Links a student with an Academic Program.</description>
+        <description>For students, their primary Program Enrollment. In Higher Ed, this typically represents a student's academic major.</description>
```
Comment (MatthewBlanski) [inline src/objects/Course_Enrollment__c.object:177]:
```suggestion
        <description>For students, their primary Program Enrollment. In Higher Ed, this typically represents a student&apos;s academic major.</description>
```

File: src/objects/Course_Enrollment__c.object [hunk 6]
```diff
@@ -158,19 +164,19 @@
...
         <externalId>false</externalId>
-        <inlineHelpText>The Program Enrollment record number for the Program the student is enrolled in. Links a student with an Academic Program.</inlineHelpText>
+        <inlineHelpText>For students, their primary Program Enrollment. In Higher Ed, this typically represents a student's academic major.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Course_Enrollment__c.object:179]:
```suggestion
        <inlineHelpText>For students, their primary Program Enrollment. In Higher Ed, this typically represents a student&apos;s academic major.</inlineHelpText>
```

File: src/objects/Course_Offering_Schedule__c.object [hunk 7]
```diff
@@ -70,9 +70,9 @@
     </fields>
     <fields>
         <fullName>End_Time__c</fullName>
-        <description>The time of day the Course Offering ends.</description>
+        <description>The time of day the Course Offering ends. If you've entered a Time Block for this Course Offering, you can leave this field blank.</description>
```
Comment (MatthewBlanski) [inline src/objects/Course_Offering_Schedule__c.object:73]:
```suggestion
        <description>The time of day the Course Offering ends. If you&apos;ve entered a Time Block for this Course Offering, you can leave this field blank.</description>
```

File: src/objects/Course_Offering_Schedule__c.object [hunk 8]
```diff
@@ -70,9 +70,9 @@
...
         <externalId>false</externalId>
-        <inlineHelpText>The time of day the Course Offering ends.</inlineHelpText>
+        <inlineHelpText>The time of day the Course Offering ends. If you've entered a Time Block for this Course Offering, you can leave this field blank.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Course_Offering_Schedule__c.object:75]:
```suggestion
        <inlineHelpText>The time of day the Course Offering ends. If you&apos;ve entered a Time Block for this Course Offering, you can leave this field blank.</inlineHelpText>
```

File: src/objects/Course_Offering_Schedule__c.object [hunk 9]
```diff
@@ -126,9 +129,9 @@
     </fields>
     <fields>
         <fullName>Start_Time__c</fullName>
-        <description>The time of day the Course Offering starts.</description>
+        <description>The time of day the Course Offering starts. If you've entered a Time Block for this Course Offering, you can leave this field blank.</description>
```
Comment (MatthewBlanski) [inline src/objects/Course_Offering_Schedule__c.object:132]:
```suggestion
        <description>The time of day the Course Offering starts. If you&apos;ve entered a Time Block for this Course Offering, you can leave this field blank.</description>
```

File: src/objects/Course_Offering_Schedule__c.object [hunk 10]
```diff
@@ -126,9 +129,9 @@
...
         <externalId>false</externalId>
-        <inlineHelpText>The time of day the Course Offering starts.</inlineHelpText>
+        <inlineHelpText>The time of day the Course Offering starts. If you've entered a Time Block for this Course Offering, you can leave this field blank.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Course_Offering_Schedule__c.object:134]:
```suggestion
        <inlineHelpText>The time of day the Course Offering starts. If you&apos;ve entered a Time Block for this Course Offering, you can leave this field blank.</inlineHelpText>
```

File: src/objects/Plan_Requirement__c.object [hunk 11]
```diff
@@ -143,9 +142,9 @@
     <fields>
         <fullName>Program_Plan__c</fullName>
         <deleteConstraint>SetNull</deleteConstraint>
-        <description>Look up field to program plan.</description>
+        <description>The Program Plan this requirement is for. If you're grouping Plan Requirements together under a Parent Plan Requirement, best practice is to link only the top-level Plan Requirement to a Program Plan.</description>
```
Comment (MatthewBlanski) [inline src/objects/Plan_Requirement__c.object:145]:
```suggestion
        <description>The Program Plan this requirement is for. If you&apos;re grouping Plan Requirements together under a Parent Plan Requirement, best practice is to link only the top-level Plan Requirement to a Program Plan.</description>
```

File: src/objects/Plan_Requirement__c.object [hunk 12]
```diff
@@ -143,9 +142,9 @@
...
         <externalId>false</externalId>
-        <inlineHelpText>Click to search for a Program Plan.</inlineHelpText>
+        <inlineHelpText>The Program Plan this requirement is for. If you're grouping Plan Requirements together under a Parent Plan Requirement, best practice is to link only the top-level Plan Requirement to a Program Plan.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Plan_Requirement__c.object:147]:
```suggestion
        <inlineHelpText>The Program Plan this requirement is for. If you&apos;re grouping Plan Requirements together under a Parent Plan Requirement, best practice is to link only the top-level Plan Requirement to a Program Plan.</inlineHelpText>
```

File: src/objects/Plan_Requirement__c.object [hunk 13]
```diff
@@ -156,9 +155,9 @@
     </fields>
     <fields>
         <fullName>Sequence__c</fullName>
-        <description>The progression of courses</description>
+        <description>This requirement's place in the Program Plan, for example, "1" indicates the first semester. Use this field to indicate the overall progression for the entire Program Plan, or to sort the order of sub-requirements within a single parent Plan Requirement.</description>
```
Comment (MatthewBlanski) [inline src/objects/Plan_Requirement__c.object:158]: Do we need to consider double quote escaping here as well?

File: src/objects/Program_Enrollment__c.object [hunk 14]
```diff
@@ -90,9 +92,9 @@
     <fields>
         <fullName>Affiliation__c</fullName>
         <deleteConstraint>Restrict</deleteConstraint>
-        <description>The Contact-Account relationship this academic information is related to.</description>
+        <description>This field is auto-populated with the Affiliation record that represents the student's enrollment in this Program. </description>
```
Comment (MatthewBlanski) [inline src/objects/Program_Enrollment__c.object:95]:
```suggestion
        <description>This field is auto-populated with the Affiliation record that represents the student&apos;s enrollment in this Program. </description>
```

File: src/objects/Program_Enrollment__c.object [hunk 15]
```diff
@@ -90,9 +92,9 @@
...
         <externalId>false</externalId>
-        <inlineHelpText>The Contact-Account relationship this academic information is related to.</inlineHelpText>
+        <inlineHelpText>This field is auto-populated with the Affiliation record that represents the student's enrollment in this Program. </inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Program_Enrollment__c.object:97]:
```suggestion
        <inlineHelpText>This field is auto-populated with the Affiliation record that represents the student&apos;s enrollment in this Program. </inlineHelpText>
```

File: src/objects/Program_Enrollment__c.object [hunk 16]
```diff
@@ -112,6 +116,8 @@
     <fields>
         <fullName>Class_Standing__c</fullName>
         <externalId>false</externalId>
+        <description>The student's level in a Program, for example, Freshman, Sophomore, Junior, or Senior.</description>
```
Comment (MatthewBlanski) [inline src/objects/Program_Enrollment__c.object:119]:
```suggestion
        <description>The student&apos;s level in a Program, for example, Freshman, Sophomore, Junior, or Senior.</description>
```

File: src/objects/Program_Enrollment__c.object [hunk 17]
```diff
@@ -112,6 +116,8 @@
...
+        <inlineHelpText>The student's level in a Program, for example, Freshman, Sophomore, Junior, or Senior.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Program_Enrollment__c.object:120]:
```suggestion
        <inlineHelpText>The student&apos;s level in a Program, for example, Freshman, Sophomore, Junior, or Senior.</inlineHelpText>
```

File: src/objects/Program_Enrollment__c.object [hunk 18]
```diff
@@ -197,6 +211,8 @@
     <fields>
         <fullName>Enrollment_Status__c</fullName>
         <externalId>false</externalId>
+        <description>The student's enrollment status, for example, full-time or half-time.</description>
```
Comment (MatthewBlanski) [inline src/objects/Program_Enrollment__c.object:214]:
```suggestion
        <description>The student&apos;s enrollment status, for example, full-time or half-time.</description>
```

File: src/objects/Program_Enrollment__c.object [hunk 19]
```diff
@@ -197,6 +211,8 @@
...
+        <inlineHelpText>The student's enrollment status, for example, full-time or half-time.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Program_Enrollment__c.object:215]:
```suggestion
        <inlineHelpText>The student&apos;s enrollment status, for example, full-time or half-time.</inlineHelpText>
```

File: src/objects/Program_Enrollment__c.object [hunk 20]
```diff
@@ -225,6 +241,8 @@
     <fields>
         <fullName>GPA__c</fullName>
         <externalId>false</externalId>
+        <description>The student's current grade point average.</description>
```
Comment (MatthewBlanski) [inline src/objects/Program_Enrollment__c.object:244]:
```suggestion
        <description>The student&apos;s current grade point average.</description>
```

File: src/objects/Program_Enrollment__c.object [hunk 21]
```diff
@@ -225,6 +241,8 @@
...
+        <inlineHelpText>The student's current grade point average.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Program_Enrollment__c.object:245]:
```suggestion
        <inlineHelpText>The student&apos;s current grade point average.</inlineHelpText>
```

File: src/objects/Program_Plan__c.object [hunk 22]
```diff
@@ -100,28 +99,28 @@
...
+        <inlineHelpText>The date this Program Plan went into effect.</inlineHelpText>
         <label>Start Date</label>
         <required>false</required>
         <trackTrending>false</trackTrending>
         <type>Date</type>
     </fields>
     <fields>
         <fullName>Status__c</fullName>
-        <description>The status of the program plan.</description>
+        <description>Select "Current" if this Program Plan has students enrolled in it, or is eligible for students to enroll. Otherwise, select "Archived."</description>
```
Comment (MatthewBlanski) [inline src/objects/Program_Plan__c.object:121]: Here again for double quotes?

File: src/objects/Relationship_Lookup__c.object [hunk 23]
```diff
@@ -16,9 +16,9 @@
     </fields>
     <fields>
         <fullName>Female__c</fullName>
-        <description>The value used if the Contact&apos;s gender field matches the values in the Female Custom Label or the Contact&apos;s salutation indicates gender.</description>
+        <description>The value used for the Reciprocal Setting if the Contact's Gender or Salutation field indicates Female gender.</description>
```
Comment (MatthewBlanski) [inline src/objects/Relationship_Lookup__c.object:19]:
```suggestion
        <description>The value used for the Reciprocal Setting if the Contact&apos;s Gender or Salutation field indicates Female gender.</description>
```

File: src/objects/Relationship_Lookup__c.object [hunk 24]
```diff
@@ -16,9 +16,9 @@
...
         <externalId>false</externalId>
-        <inlineHelpText>The value used if the Contact&apos;s gender field matches the values in the Female Custom Label or the Contact&apos;s Salutation indicates gender.</inlineHelpText>
+        <inlineHelpText>The value used for the Reciprocal Setting if the Contact's Gender or Salutation field indicates Female gender.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Relationship_Lookup__c.object:21]:
```suggestion
        <inlineHelpText>The value used for the Reciprocal Setting if the Contact&apos;s Gender or Salutation field indicates Female gender.</inlineHelpText>
```

File: src/objects/Relationship_Lookup__c.object [hunk 25]
```diff
@@ -28,9 +28,9 @@
     </fields>
     <fields>
         <fullName>Male__c</fullName>
-        <description>The value used if the Contact&apos;s gender field matches the values in the Male Custom Label or the Contact&apos;s salutation indicates gender.</description>
+        <description>The value used for the Reciprocal Setting if the Contact's Gender or Salutation field indicates Male gender.</description>
```
Comment (MatthewBlanski) [inline src/objects/Relationship_Lookup__c.object:31]:
```suggestion
        <description>The value used for the Reciprocal Setting if the Contact&apos;s Gender or Salutation field indicates Male gender.</description>
```

File: src/objects/Relationship_Lookup__c.object [hunk 26]
```diff
@@ -28,9 +28,9 @@
...
         <externalId>false</externalId>
-        <inlineHelpText>The value used if the Contact&apos;s gender field matches the values in the Male Custom Label or the Contact&apos;s Salutation indicates gender.</inlineHelpText>
+        <inlineHelpText>The value used for the Reciprocal Setting if the Contact's Gender or Salutation field indicates Male gender.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Relationship_Lookup__c.object:33]:
```suggestion
        <inlineHelpText>The value used for the Reciprocal Setting if the Contact&apos;s Gender or Salutation field indicates Male gender.</inlineHelpText>
```

File: src/objects/Relationship__c.object [hunk 27]
```diff
@@ -134,12 +136,12 @@
     <fields>
         <fullName>Relationship_Explanation__c</fullName>
         <description>Text formula that constructs a sentence explaining the relationship between the two Contacts</description>
+        <inlineHelpText>A sentence that clarifies who's who in the Relationship. For example, &quot;&apos;s &quot; Joe Ada is Sara Cordova's employer.&quot;&apos;s &quot; This field is read-only.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Relationship__c.object:139]:
```suggestion
        <inlineHelpText>A sentence that clarifies who&apos;s who in the Relationship. For example, &quot;&apos;s &quot; Joe Ada is Sara Cordova's employer.&quot;&apos;s &quot; This field is read-only.</inlineHelpText>
```

File: src/objects/Term__c.object [hunk 28]
```diff
@@ -80,14 +82,17 @@
...
+        <inlineHelpText>The date this Term ends.</inlineHelpText>
         <label>End Date</label>
         <required>false</required>
         <trackTrending>false</trackTrending>
         <type>Date</type>
     </fields>
     <fields>
         <fullName>Grading_Period_Sequence__c</fullName>
-        <description>Such as 1 for the first six week, 2 for the second six weeks, etc.</description>
+        <description>Indicates which Term this is. For example, enter "1"  for the first quarter or semester, "2" for the second quarter or semester, etc.</description>
```
Comment (MatthewBlanski) [inline src/objects/Term__c.object:94]: Here again for double quotes?

File: src/objects/Trigger_Handler__c.object [hunk 29]
```diff
@@ -202,16 +203,16 @@
     <fields>
         <fullName>User_Managed__c</fullName>
         <defaultValue>false</defaultValue>
-        <description>The record is managed by the user and updates won&apos;t be pushed to it.</description>
+        <description>When selected, we no longer push updates to this Trigger Handler. Select this option if you've made changes that you don't want us to overwrite.</description>
```
Comment (MatthewBlanski) [inline src/objects/Trigger_Handler__c.object:206]:
```suggestion
        <description>When selected, we no longer push updates to this Trigger Handler. Select this option if you&apos;ve made changes that you don't want us to overwrite.</description>
```

File: src/objects/Trigger_Handler__c.object [hunk 30]
```diff
@@ -202,16 +203,16 @@
...
         <externalId>false</externalId>
-        <inlineHelpText>Check it if you intent to manage the record yourself. Be aware that updates to the package won&apos;t update the record.</inlineHelpText>
+        <inlineHelpText>When selected, we no longer push updates to this Trigger Handler. Select this option if you've made changes that you don't want us to overwrite.</inlineHelpText>
```
Comment (MatthewBlanski) [inline src/objects/Trigger_Handler__c.object:208]:
```suggestion
        <inlineHelpText>When selected, we no longer push updates to this Trigger Handler. Select this option if you&apos;ve made changes that you don't want us to overwrite.</inlineHelpText>
```

File: src/objects/Plan_Requirement__c.object (hunk 13 above)
Comment (lifewithryan) [reply src/objects/Plan_Requirement__c.object:158]: we haven't been that I am aware of.

**Comment by dgamboni** [comment]
Tested/Passes QE. See Work Item for add'l notes.

**Pull request merged**